gapis/gfxapi/templates/api_types.h.tmpl

```diff
@@ -64,6 +64,9 @@
   {{range $p := $.Pseudonyms}}
     {{Template "DeclareType" $p}}
   {{end}}
+  {{range $c := $.Classes}}
+    {{if GetAnnotation $c "internal"}} class {{$c.Name}}; {{end}}
+  {{end}}
```

Comment:
@ben-clayton @AWoloszyn Not sure if this is acceptable, but this does solve the undefined-type problem when compiling CPP